client/disputes/index.tsx

```diff
@@ -37,18 +38,22 @@ import { getPostUrl } from 'wcpay/utils';
 
 import './style.scss';
 
-const headers: DisputesTableHeader[] = [
+const getHeaders = ( sortByAmount: boolean ): DisputesTableHeader[] => [
```

Comment:
This approach is not extensible if we want to enable sorting for more columns. Instead, we should pass here the sorting column name and change the `cellClassName` generation where needed.

Reply:
I am not sure if I follow you. Let me explain the reason why I added the `sortByAmount ` flag n case it helps. This flags tries to solve a visual issue. It will add the 'is-sorted' class to the first column of the table, only when the table is sorted by `amount`. This gives it a nicer appearance, as if both columns were the same, and not two different ones. I saw that we are doing the same in Transactions, Deposits...

Maybe an image is worth a thousand words in this case. This is the visual issue it tries to solve:

![Screen Shot 2022-01-20 at 10 48 25](https://user-images.githubusercontent.com/1553182/150314586-4cf0e57e-10b4-42e3-bd08-9d0bd647af85.png)
 
vs.

![Screen Shot 2022-01-20 at 10 50 40](https://user-images.githubusercontent.com/1553182/150314913-fd7953d9-ddbc-4493-85cf-a93c98633944.png)

![Screen Shot 2022-01-20 at 10 51 40](https://user-images.githubusercontent.com/1553182/150315137-e4b22552-5ef9-4baf-b5b3-dd71163230ec.png)

In my understanding, this does not affect if we want to enable sorting for more columns in the future. I am afraid that maybe we would add more complexity that is not needed in this case? Please let me know if I am missing something.

Reply:
Something like this:

```typescript
const getHeaders = ( sortColumn: string ): DisputesTableHeader[] => [
```

```typescript
        cellClassName: classNames( 'info-button', {
			'is-sorted': sortColumn === 'column-name-here',
		} ),
```

Reply:
That's not much added complexity, but a timely generalization.

Reply:
Done, thanks!